### PR TITLE
Update Treehouse

### PR DESCRIPTION
**Pull request by fbernhart**
treehouse

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

File: icons/treehouse.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...62-.9168-.5677-1.4727-.5005-.557-1.5232.4945-1.8536 1.5925l-.0158.0603c-.3304 1.082-.54 2.0277-.4768 2.1197.0366.0593.0712.1187.1039.18A1.7905 1.7905 0 019.549 18.752a1.7864 1.7864 0 01.7388-2.4194l.1197-.0603c.0643-.0297.2572-.4808.4184-1.0208l.2977-.9763c.0188-.0465.0327-.1058.0445-.1661l.8873-3.1553-.9466 1.9386c-.1108-.359-.2967-.3758-.7013-.09-.2403.181-.6459.5253-.8407.6617-.361.27-.6776.7795-.8418 1.097a1.2987 1.2987 0 01-.4055.4945c-.6766.558-1.6687.451-2.2097-.2245a1.5737 1.5737 0 01.902-2.5242c.4046-.091 1.7577-.917 2.5678-1.5035.1504-.1048.271-.1948.3759-.3007l2.179-1.6518-1.8932 1.0208s-.1938-.0159-.4213.0168c-.6904.0742-1.4125.18-1.547.3462a.9881.9881 0 01-.2868.269 1.4147 1.4147 0 01-1.9674-.3135 1.4214 1.4214 0 01.3323-1.9723c.2562-.181.5708-.271.8556-.271.5124.0158 1.8497.2868 2.9605.091l.2848-.0446c1.1276-.1938 2.449-1.173 2.9911-2.1632l1.1711-2.0445c.646-1.1266.6014-2.358-.1038-2.7646L13.2018.2693c-.646-.359-1.6964-.359-2.3284 0l-8.7616 4.912C1.452 5.5265.9258 6.4275.9258 7.1644v9.638c0 .736.5262 1.638 1.1652 1.998l8.7537 4.9288c.645.361 1.6973.361 2.3284 0l8.73-4.9288c.6459-.364 1.171-1.262 1.171-1.998V7.1486c0-.7359-.5251-1.635-1.1562-1.998l-1.4441-.814"/></svg>
```
Comment (adamrusted) [inline icons/treehouse.svg:1]: Looks perfect! 🎉
Any chance of getting it down to 3 decimals?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/treehouse.svg (hunk 1 above)
Comment (fbernhart) [reply icons/treehouse.svg]: Done 💪🏼

File: icons/treehouse.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...449-2.07.275-2.84 1.617l-1.4 2.435a3.07 3.07 0 0 0 .207 3.145l.041.062c.646.93 1.518 1.788 1.803 2.095a1.595 1.595 0 0 1-.73 2.625 1.586 1.586 0 0 1-2.004-1.728c.045-.346-.066-.917-.567-1.473-.501-.557-1.524.495-1.854 1.592l-.016.06c-.33 1.083-.54 2.028-.477 2.12.037.06.072.12.104.18a1.79 1.79 0 0 1-3.161 1.683 1.786 1.786 0 0 1 .739-2.42l.12-.06c.064-.03.257-.48.418-1.02l.298-.977a.96.96 0 0 0 .044-.166l.887-3.155-.946 1.938c-.111-.359-.297-.375-.701-.09-.24.181-.646.526-.841.662-.361.27-.678.78-.842 1.097a1.299 1.299 0 0 1-.406.495 1.558 1.558 0 0 1-2.21-.225 1.574 1.574 0 0 1 .903-2.524c.404-.091 1.758-.917 2.568-1.504.15-.104.27-.194.375-.3l2.18-1.652-1.894 1.02s-.194-.015-.421.017c-.69.075-1.412.18-1.547.347a.988.988 0 0 1-.287.269 1.415 1.415 0 0 1-1.967-.314 1.421 1.421 0 0 1 .332-1.972c.256-.181.57-.271.856-.271.512.016 1.85.287 2.96.09l.285-.044c1.128-.194 2.45-1.173 2.991-2.163l1.171-2.044c.646-1.127.602-2.358-.104-2.765L13.202.269c-.646-.359-1.697-.359-2.329 0L2.112 5.181C1.452 5.526.926 6.428.926 7.164v9.638c0 .736.526 1.638 1.165 1.998l8.754 4.93c.645.36 1.697.36 2.328 0l8.73-4.93c.646-.364 1.171-1.262 1.171-1.998V7.15c0-.736-.525-1.635-1.156-1.998l-1.444-.814"/></svg>
```
Comment (adamrusted) [inline icons/treehouse.svg:1]: It looks like they've now smoothed out the logo a little bit - might need to adjust the SVG...
```svg
M20.398,4.386c-0.821-0.448-2.09,0.299-2.837,1.642l-1.418,2.389c-0.523,0.97-0.448,2.165,0.224,3.135l0.075,0.075 	c0.672,0.896,1.493,1.792,1.792,2.09c0.149,0.149,0.299,0.373,0.373,0.672c0.224,0.821-0.224,1.717-1.12,1.941 	c-0.821,0.224-1.717-0.224-1.941-1.12c-0.075-0.224-0.075-0.448-0.075-0.597c0.075-0.373-0.075-0.896-0.597-1.493 	c-0.523-0.523-1.493,0.523-1.866,1.568v0.075c-0.299,1.045-0.523,2.016-0.448,2.09c0,0.075,0.075,0.075,0.075,0.149 	c0.448,0.896,0.149,1.941-0.747,2.389s-1.941,0.149-2.389-0.747c-0.448-0.896-0.149-1.941,0.747-2.389 	c0.075,0,0.075-0.075,0.149-0.075c0.075,0,0.224-0.448,0.448-1.045c0.149-0.523,0.224-0.747,0.299-0.97 	c0.075-0.299,0.224-1.045,0.149-1.344c-0.075-0.373-0.448-0.373-0.821-0.149c-0.224,0.149-0.672,0.523-0.821,0.672 	c-0.373,0.299-0.672,0.747-0.821,1.12c-0.075,0.149-0.224,0.373-0.373,0.523c-0.672,0.523-1.642,0.448-2.24-0.224 	c-0.523-0.672-0.448-1.642,0.224-2.24c0.224-0.149,0.448-0.299,0.672-0.299c0.373-0.075,1.717-0.896,2.538-1.493 	c0.149-0.075,0.373-0.299,0.448-0.373c0.224-0.299,0-0.523-0.224-0.523C9.201,9.91,8.454,9.984,8.305,10.208 	c-0.075,0.075-0.149,0.224-0.299,0.299c-0.597,0.448-1.493,0.299-1.941-0.299C5.617,9.611,5.767,8.715,6.364,8.267 	c0.224-0.224,0.597-0.299,0.821-0.299c0.523,0,1.866,0.299,2.911,0.075l0.299-0.075c1.12-0.224,2.463-1.194,2.986-2.165 	c0,0,0.523-0.896,1.12-2.016c0.672-1.12,0.597-2.389-0.075-2.762L13.157,0.28c-0.672-0.373-1.642-0.373-2.314,0L2.184,5.207 	C1.512,5.58,0.989,6.476,0.989,7.222v9.555c0,0.747,0.523,1.642,1.194,2.016l8.659,4.927c0.672,0.373,1.717,0.373,2.314,0 	l8.659-4.927c0.672-0.373,1.194-1.269,1.194-2.016V7.222c0-0.747-0.523-1.642-1.194-2.016C21.816,5.207,21.219,4.834,20.398,4.386z
```

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/treehouse.svg (hunk 2 above)
Comment (fbernhart) [reply icons/treehouse.svg]: @adamrusted Done. A precision of 3 didnt't make it through the linter, so I had to increase it to 4.

**Review by adamrusted — APPROVED** [review]
LGTM! 🐸

**Pull request merged**